Numsense.UnitTests/AssemblyInfo.fs

```diff
@@ -34,8 +34,8 @@ open System.Runtime.InteropServices
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [<assembly: AssemblyVersion("1.0.*")>]
-[<assembly: AssemblyVersion("0.4.0.0")>]
-[<assembly: AssemblyFileVersion("0.4.0.0")>]
+[<assembly: AssemblyVersion("0.5.0.0")>]
+[<assembly: AssemblyFileVersion("0.5.0.0")>]
```

Comment:
Please don't change the version. There's no guarantee that the previous version is still 0.4.0 when this pull request is merged. It'll depend on which other changes were made before this happens.

[I'll update the version number appropriately before releasing a new version](http://blog.ploeh.dk/2013/12/10/semantic-versioning-with-continuous-deployment).